exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/indices/StoragePreconditions.java

```diff
@@ -138,6 +138,20 @@ static long checkPositionIndex(long index, long size) {
     return index;
   }
 
+  /**
+   * Checks that the first element index of the range is less than or equal
+   * to the index after the last element.
+   *
+   * @param from the index of the first element
+   * @param to the index after the last element
```

Comment:
inclusive/exclusive?

Reply:
*after the last* implies exclusive (after the last in the range)